lib/metadata/VmConfig/xmlConfig.rb

```diff
@@ -12,9 +12,9 @@ def convert(filename)
       if Sys::Platform::IMPL == :linux
         begin
           # First check to see if the command is available
-          MiqUtil.runcmd("virsh list")
+          AwesomeSpawn.run!("virsh", :params => ["list"])
```

Comment:
Note that unlike the other conversions the :combined_output, and .output are not needed here, since the output is ignored anyway.  All that is cared about here is whether the command blows up or not, which is covered by `run!`